### PR TITLE
Now it works without ACML

### PR DESCRIPTION
**Pull request by ivashnyov**
Made it works without depricated ACML library

### CODE REVIEW

File: textons/filters.cpp [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #include <stdio.h>
 #include <math.h>
 #include <complex>
-//#include "filters.h"
```
Comment (amueller) [inline textons/filters.cpp]: I don't understand this?

**Comment by amueller** [comment]
so you are just vendoring the required files? Is there not another library providing these?

File: textons/filters.cpp (hunk 1 above)
Comment (ivashnyov) [reply textons/filters.cpp]: This is an artifact from trying to make it work

**Comment by ivashnyov** [comment]
Unfortunately, I could not find one. Moreover, this method allows not to depend on libraries.

**Comment by amueller** [comment]
ok, seems reasonable. Sorry for the slow reply. Should I merge?

**Comment by ivashnyov** [comment]
I think yes :)

**Comment by amueller** [comment]
Thanks!

**Pull request merged**